catalog/static/config.js

```diff
@@ -1,12 +1,14 @@
 /* eslint-disable */
-if (window.location.hostname === 'quiltdata.com') {
+const { hostname } = window.location;
```

Comment:
This file doesn't get processed by the ES6 compiler, so I don't think we can do this here - unless it's supported by all browsers we care about.